docs/content/preview/explore/ysql-language-features/going-beyond-sql/follower-reads-ysql.md

```diff
@@ -26,12 +26,13 @@ type: docs
   </li>
 </ul>
 
-## Leader leases
+## Leaders and leader leases
 
-In a distributed environment, when one node in a cluster is elected as the leader holding the latest data, it is possible that another node may assume that it is the leader, and that it holds the latest data. This could result in serving stale reads to a client. To avoid this confusion, YugabyteDB provides a _leader lease_ mechanism where an elected node member is guaranteed to be the leader until its lease expires.
+In a distributed environment, when one node in a cluster is elected as the leader holding the latest data, it is possible that another node may assume that it is the leader, and that it holds the latest data. This could result in serving stale reads to a client. To avoid this confusion, YugabyteDB provides a [leader lease mechanism](../../../../architecture/transactions/single-row-transactions/) where an elected node member is guaranteed to be the leader until its lease expires.
```

Comment:
```suggestion
In a distributed environment, when one node in a cluster is elected as the leader holding the latest data, it is possible that another node may assume that it is the leader, and that it holds the latest data. This could result in serving stale reads to a client. To avoid this confusion, YugabyteDB provides a [leader lease mechanism](../../../../architecture/transactions/single-row-transactions/#leader-leases-reading-the-latest-data-in-case-of-a-network-partition) where an elected node member is guaranteed to be the leader until its lease expires.
```

Reply:
I would say just link to leader lease mechanism directly.